src/metpy/units.py

```diff
@@ -82,6 +82,7 @@ def setup_registry(reg):
                '= degreeN')
     reg.define('degrees_east = degree = degrees_E = degreesE = degree_east = degree_E '
                '= degreeE')
+    reg.define('dBZ = 1 ; logbase: 10; logfactor: 10')
```

Comment:
Let's switch to this for better abbreviations and support for correct physical units in linear space:
```suggestion
    reg.define('dBz = 1e-18 m^3; logbase: 10; logfactor: 10 = dBZ')
```